cfg/builder/builder_walk.cc

```diff
@@ -675,7 +675,9 @@ BasicBlock *CFGBuilder::walk(CFGContext cctx, ast::Expression *what, BasicBlock
             [&](ast::Cast *c) {
                 core::LocalVariable tmp = cctx.newTemporary(core::Names::castTemp());
                 current = walk(cctx.withTarget(tmp), c->arg.get(), current);
-                current->exprs.emplace_back(cctx.target, c->loc, make_unique<Cast>(tmp, c->type, c->cast));
+                if (c->cast != core::Names::uncheckedLet()) {
```

Comment:
Will anything fail if `cctx.target` doesn't get populated? Could you add cfg exp tests for some of the `T::Enum` tests and make sure that everything makes sense?